vnext/docs/ProjectStructure.md

```diff
@@ -0,0 +1,100 @@
+# MSBuild Project Structure
+
+This is a description of each of the core projects, their purpose and output artifacts.<br/>
+Sample applications are not covered.
+
+## Alphabetical Index
+- [Chakra\Chakra.vcxitems](#Chakra)
+- [Desktop\React.Windows.Desktop.vcxproj](#React.Windows.Desktop)
+- **[Desktop.DLL\React.Windows.Desktop.DLL.vcxproj](#React.Windows.Desktop.DLL)**
+- [Desktop.IntegrationTests\React.Windows.Desktop.IntegrationTests.vcxproj](#React.Windows.Desktop.IntegrationTests)
+- [Desktop.UnitTests\React.Windows.Desktop.UnitTests.vcxproj](#React.Windows.Desktop.UnitTests)
+- [Folly\Folly.vcxproj](#Folly)
+- [FollyWin32\FollyWin32.vcxproj](#FollyWin32)
+- [IntegrationTestScripts\IntegrationTests.njsproj](#IntegrationTests-Node-Project)
+- [IntegrationTests\React.Windows.IntegrationTests.vcxproj](#React.Windows.IntegrationTests)
+- [ReactCommon\ReactCommon.vcxproj](#ReactCommon)
+- **[ReactUWP\ReactUWP.vcxproj](#ReactUWP)**
+- [ReactWindowsCore\ReactWindowsCore.vcxproj](#ReactWindowsCore)
+- [Shared\Shared.vcxitems](#Shared)
+- [Universal.IntegrationTests\React.Windows.Universal.IntegrationTests.vcxproj](#React.Windows.Universal.IntegrationTests)
+- [Universal.UnitTests\React.Windows.Universal.UnitTests.vcxproj](#React.Windows.Universal.UnitTests)
+
+## Common Projects
+
+### ReactWindowsCore
+*Static Library*<br/>
+Contains common functionality for both Desktop and Universal Windows variants.
+
+### Shared
+*Shared Items (no build artifact)*<br/>
+Holds sources common to both Windows variants, that require different build configuration
+(i.e. compiler flags, language standard).
+
+### ReactCommon
```

Comment:
>ReactCommon [](start = 4, length = 11)

Are the contents of this project "links" to sources in a FB package? In other words, can we add or change the contents of this project, or should this be read-only?

Reply:
If you mean sources like ReactCommon and Folly, they are fetched as npm dependencies.
Changes on them will not be persisted anywhere so should be treated as read-only, but can be edited for experimentation purposes.